client/my-sites/design-preview/index.js

```diff
@@ -13,11 +13,12 @@ import url from 'url';
  */
 import config from 'config';
 import WebPreview from 'components/web-preview';
-import * as PreviewActions from 'state/preview/actions';
+import { clearPreviewSlug } from 'state/ui/actions';
+import { fetchPreviewMarkup, undoCustomization, clearCustomizations } from 'state/preview/actions';
```

Comment:
Glad to see this change. May not have been the intent, but this will help for tree shaking [when we get to that point](http://www.2ality.com/2015/12/webpack-tree-shaking.html). In general, I think `import * as` should be avoided as much as possible, as it can't be optimized.